README.txt

```diff
@@ -58,30 +87,28 @@ some data for testing and some simulator outputs]
 	INCLUDECADMIUM=-I ../../cadmium/include
     Update the relative path to cadmium/include from the folder where the makefile is. You need to take into account where you copied the folder during the installation process
 	Example: INCLUDECADMIUM=-I ../../cadmium/include
-    
-2 - Run the unit tests
-	2.1. Run subnet test
-		1 - Open the terminal. Press in your keyboard Ctrl+Alt+t
-		2 - Set the command prompt in the test/subnet folder. To do so, type in the terminal the path to this folder.
-			Example: cd Documents/AlternateBitProtocol/test/subnet
-		3 - To compile the test, type in the terminal:
-			make clean; make all
-		4 - To run the test, type in the terminal "./NAME_OF_THE_COMPILED_FILE". For this specific test you need to type:
-			./SUBNET
-		5 - To check the output of the test, open  "subnet_test_output.txt"
-	2.2. To run receiver and sender tests, the steps are analogous to 2.1
 			
-3 - Run the simulator
+2 - Run the simulator and unit tests
 	1 - Open the terminal. Press in your keyboard Ctrl+Alt+t
-	2 - Set the command prompt in the top_model folder. To do so, type in the terminal the path to this folder.
-		Example: cd ../../top_model
-	3 - To compile the project, type in the terminal:
+	2 - Set the command prompt in the AlternateBitProtocolSimulator folder. To do so, type in the terminal the path to this folder.
+		Example: cd ../../AlternateBitProtocolSimulator
+	3 - To compile the whole project, type in the terminal:
 		make clean; make all
 	4 - To run the simulation, type in the terminal "./NAME_OF_THE_COMPILED_FILE NAME_OF_THE_INPUT_FILE". For this test you need to type:
-		./ABP input_abp_1.txt
-	5 - To check the output of the simulation, open  "abp_output.txt"
+		./bin/top_model/ABP data/top_model/input_abp_1.txt
+	5 - To check the output of the simulation, open  "abp_output.txt" present in data/top_model folder.
 	6 - To execute the simulator with different inputs
 		6.1. Create new .txt files with the same structure as input_abp_0.txt or input_abp_1.txt
 		6.2. Run the simulator using the instructions in step 4
 		6.3. If you want to keep the output, rename abp_output.txt. To do so, type in the terminal: "mv abp_output.txt NEW_NAME"
-			Example: mv abp_output.txt abp_output_0.txt
+			Example: mv data/top_model/abp_output.txt data/top_model/abp_output_0.txt
+	7 - To run the subnet test, type in the terminal "./NAME_OF_THE_COMPILED_FILE". For this specific test you need to type:
+			./bin/test/SUBNET_TEST

```

Comment:
Remove tab or 4 spaces as it should be in line with 7.1

Reply:
resolved the indentation issue